search/src/Components/upload/uploader_zip.js

```diff
@@ -1,61 +1,131 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+
 import 'react-dropzone-uploader/dist/styles.css';
 import Dropzone from 'react-dropzone-uploader';
 import Cookies from 'js-cookie';
 
+import axios from 'axios';
+
+import Uppy from '@uppy/core';
+import '@uppy/core/dist/style.css';
+import '@uppy/dashboard/dist/style.css';
+import Tus from '@uppy/tus';
+import { Dashboard, useUppy } from '@uppy/react';
+
+import {jsonSchemaTitle} from '../error/utils';
+
+const baseURL = new URL(window.location.origin);
+
+
+const TUS_ENDPOINT = baseURL + 'tus/files/';
+const TUS_CHUNK_SIZE = 1024 * 1024 * 10;
 
-const UploaderZip  = props => {
-    const baseURL = new URL(window.location.origin);
-    const getUploadParams = ({ file, meta }) => {
-        const body = new FormData()
-        body.append('upload_id', props.upload_id)
-        body.append('images', props.images)
-        body.append('upload_image_zip', file)
-        return { url: baseURL + 'api/upload_image_zip/',
-                 mode: 'same-origin',
-                 headers: {'X-CSRFToken': Cookies.get('csrftoken')},
-                 body
-               }
+
+function getTusUploadFile(file) {
+    const url = file.tus.uploadUrl;
```

Comment:
Have we mostly used a 4x or a 2x indent in JS? We might consider adopting Prettier.js or similar...

Reply:
Having a browse around, it looks like about half-and-half - I was following the original upload_zip.js and upload_zip_images.js